### DISCUSSION TITLE
React-Native and Saleor

### DISCUSSION THREAD

**Discussion by Mansi-Bigstep** [saleor/saleor#9022] [Q&A]
How can we create react native app using saleor sdk ?

**Comment by karolkielecki** [comment]
Hi @Mansi-Bigstep,

Saleor SDK is not compatible with React Native, but that's our plan.

**Reply by Mansi-Bigstep** [reply]
When can we expect it to be compatible? Also, can u suggest any way to make saleor work with React Native?

**Reply by karolkielecki** [reply]
There is no ETA, so can't promise any date. Maybe you can find something useful here: https://saleor.io/blog/creating-a-mobile-application-with-react-native-using-the-saleor-graphql-api-144/

**Reply by Mansi-Bigstep** [reply]
I tried this but when I connect this to my local graphql and my hosted API on heroku it doesn't work. The API is working fine when connected wth storefront. Can u suggest something in case I am missing anything while creating apolloclient maybe? Does it require anything else apart from the apiurl of graphql maybe token or something?

**Comment by AnbazhaganVenkatesan** [comment]
Hi @karolkielecki  any update on this ? 
Saleor SDK is not compatible with React Native, but that's our plan.
